migration_original/ODS1Stage/tables/Show/SOLRMarket/spu_translated_SOLRMarket.sql

```diff
@@ -0,0 +1,453 @@
+CREATE OR REPLACE PROCEDURE SP_LOAD_TABLE_SOLRMarket() -- Parameters
+    RETURNS STRING
+    LANGUAGE SQL
+    EXECUTE AS CALLER
+    AS  
+
+DECLARE 
+
+---------------------------------------------------------
+--------------- 0. Table dependencies -------------------
+---------------------------------------------------------
+
+--- Mid.ClientMarket
+--- Base.ClientEntityToClientFeature
+--- Base.EntityType
+--- Base.ClientFeatureToClientFeatureValue
+--- Base.ClientFeature
+--- Base.ClientFeatureValue
+--- Base.ClientFeatureGroup
+--- Base.vwuCallCenterDetails
+--- Base.Market
+
+---------------------------------------------------------
+--------------- 1. Declaring variables ------------------
+---------------------------------------------------------
+
+    select_statement STRING; -- CTE and Select statement for the Merge
+    update_statement STRING; -- Update statement for the Merge
+    insert_statement STRING; -- Insert statement for the Merge
+    merge_statement STRING; -- Merge statement to final table
+    status STRING; -- Status monitoring
+   
+---------------------------------------------------------
+--------------- 2.Conditionals if any -------------------
+---------------------------------------------------------   
+   
+BEGIN
+
+---------------------------------------------------------
+----------------- 3. SQL Statements ---------------------
+---------------------------------------------------------     
+
+--- Select Statement
+-- If no conditionals:
+select_statement := '
+WITH cte_mid AS (
```

Comment:
Remove the Limit 100 and add a comment here explaining the reason for this cte (to allow batch processing).